src/frequenz/channels/util/_file_watcher.py

```diff
@@ -26,6 +27,15 @@ class EventType(Enum):
         MODIFY = Change.modified
         DELETE = Change.deleted
 
+    @dataclass(frozen=True)
+    class Event:
+        """A class to notify events when a file changes."""
```

Comment:
I don't think events need to be modified :)
Maybe just "Class used to notify about file changes"?

Reply:
Sounds good, will update it

Reply:
I don't get this, I don't read from there that events need to be modified. But now that you are moving my attention there, I think we should avoid starting class documentation with "A class to" and just say "A file change event." or something like that.

Reply:
I didn't see the last comment. @Marenz  does `A file change event.`  sound go to you?